tools/azsdk-cli/docs/specs/0-scenario-1.spec.md

```diff
@@ -379,41 +378,43 @@ Updating package metadata for Health Deidentification SDK...
 Package metadata updated for 5/5 languages
 ```
 
-### 4. Run PR Checks
+### 4. Run Validation Checks
 
-> Sample: will be changed when the spec for this tool is added
 
 **Command:**
 ```bash
-azsdk run-pr-checks --service healthdataaiservices --languages .NET,Java,JavaScript,Python,Go
+# Run all checks (default)
+azsdk package run-checks --package-path <path_to_sdk_package>/
+
+# Run specific check type
+azsdk package run-checks <check-type> --package-path <path_to_sdk_package>/
 ```
 
 **Options:**
-- `--service <name>`: Service name to check (required)
-- `--languages <list>`: Languages to check (default: all)
-- `--checks <list>`: Specific checks to run (build,test,lint,breaking-changes)
-- `--parallel`: Run checks in parallel
-- `--fail-fast`: Stop on first failure
+- `--package-path <path>`: Path to the specific SDK package directory (required)
+
+**Available Check Types (as subcommands):**
+- `all`: Run all available validation checks (default)
+- `changelog`: Run changelog validation check
+- `dependency`: Run dependency analysis check  
+- `readme`: Run README validation check
+- `cspell`: Run spelling validation check
+- `snippets`: Run snippet update check
+- `linting`: Run code linting check
+- `format`: Run code formatting check
+- `checkaotcompat`: .NET AOT compatibility validation
+- `generatedcodechecks`: .NET generated code validation
```

Comment:
Just curious, how is this different from other checks ?

Reply:
is it only for generated code? or for custom code too.
(are there any document or settings checks)?

Reply:
It regenerates code, exports the API, checks the solution file is correct, re-generates the snippets and does additional .NET specific README validations beyond the common script

Reply:
From above what you noted @m-redding I do feel this is more like a release-readiness check than a validation check then?
Also the current naming makes it seem focused only for generated code?

Reply:
It does more than check generated code but code checks felt too vague to me. I'm not stuck on that name if someone has a better idea

Reply:
`dotnetchecks` ?

Reply:
@l0lawrence feel free to add them individually for now, or separate them into a standard design for the languages

Reply:
I think for .NET this actually relates to the longer discussion here - https://github.com/Azure/azure-sdk-tools/pull/12563#discussion_r2449474956

My suggestion is leaving this as-is (or doing the language thing - I don't have a preference) for the first scenario. I would like to do some work to adjust/split up a lot of these scripts that we have in the .NET repo to be more friendly for running in multiple contexts (local, cli, pipelines, etc). Because right now our scripts are really written only for the pipelines. But I would prefer to do this at the same time as any work from the outcome of the discussion above if possible.

Reply:
I addressed this already above. I did not write this script and I'm not going to rewrite any scripts until we have alignment on https://github.com/Azure/azure-sdk-tools/pull/12563#discussion_r2449474956